src/robot/model/testsuite.py

```diff
@@ -50,12 +57,12 @@ def __init__(self, name: str = '', doc: str = '', metadata: dict = None,
         self.rpa = rpa        #: ``True`` when RPA mode is enabled.
         self.suites = None
         self.tests = None
-        self._setup = None
-        self._teardown = None
+        self._setup: Union[Keyword, None] = None
+        self._teardown: Union[Keyword,  None] = None
```

Comment:
Not part of the public API so type hints probably aren't needed.

Reply:
Removed